File: app/src/main/java/br/com/patrocine/patrocine/model/response/MovieResponse.java
package br.com.patrocine.patrocine.model.response;

import com.google.gson.annotations.SerializedName;
import br.com.patrocine.patrocine.model.Movie;

import java.util.ArrayList;

public class MovieResponse {

    @SerializedName("movies")
    private ArrayList<Movie> results;


    public MovieResponse(ArrayList<Movie> results){
        setResults(results);
    }

    public ArrayList<Movie> getResults(){
        return results;
    }

    public void setResults(ArrayList<Movie> results){
        this.results = results;
    }

}

File: app/src/main/java/br/com/patrocine/patrocine/model/response/MovieResponse.kt
package br.com.patrocine.patrocine.model.response

import br.com.patrocine.patrocine.model.Movie
import com.google.gson.annotations.SerializedName
import java.util.ArrayList

class MovieResponse {
    @SerializedName("movies")
    var results: ArrayList<Movie>? = null
}